snapcraft/internal/errors.py

```diff
@@ -166,13 +166,26 @@ class InvalidAppCommandError(SnapcraftError):
 
     fmt = (
         "Failed to generate snap metadata: "
-        "The specified command {command!r} defined in the app {app!r} does "
+        "The specified command {command!r} defined in the app {app_name!r} does "
         "not exist or is not executable.\n"
         "Ensure that {command!r} is relative to the prime directory."
     )
 
-    def __init__(self, command, app):
-        super().__init__(command=command, app=app)
+    def __init__(self, command, app_name):
```

Comment:
:+1:

Reply:
maybe force kwargs with *?

Reply:
That doesn't really match the pattern established here, I just started getting confused with what "app" meant, so I wanted to change it to "app_name" :stuck_out_tongue: .